Benkimeric/PBP-Test
# Project setup and components structuring

#### What does this PR do?
- Implements initial setup for the project and component design
#### Description of Task to be completed?
- [x] add basic components
- [x] install required libraries
- [x] build basic structure for the app

#### How should this be manually tested?
- Clone this repo `git clone  [https://github.com/Benkimeric/PBP-Test.git]`
- Checkout to this branch `ch-setup`
- Install all dependencies `yarn`
- Run  `yarn start`
- Notice the layout for the application according to design

#### Any background context you want to provide?

#### Screenshots (if appropriate)
http://g.recordit.co/davwkEMrlS.gif
#### Questions:
